cmd/sourced/compose/compose.go

```diff
@@ -40,6 +40,10 @@ func (c *Compose) RunWithIO(ctx context.Context, stdin io.Reader,
 		return err
 	}
 
+	if err := workdir.ValidateWorkdir(dir); err != nil {
```

Comment:
nit pic: `workdir.ValidateWorkdir` -> `workdir` repeated 2 times in the name of package an in the name of function.